test/k8sT/hubble.go

```diff
@@ -198,6 +198,20 @@ var _ = Describe("K8sHubbleTest", func() {
 			res.ExpectContains(`hubble_flows_processed_total{protocol="TCP",subtype="to-endpoint",type="Trace",verdict="FORWARDED"}`)
 		})
 
+		It("Test TLS certificate", func() {
+			cmd := "cat /var/lib/cilium/tls/hubble/server.crt"
+			res := kubectl.ExecPodCmd(helpers.CiliumNamespace, ciliumPodK8s1, cmd)
+			res.ExpectSuccess("%q cannot read the hubble server TLS certificate", ciliumPodK8s1)
+			expected := string(res.GetStdOut().Bytes())
+
+			nodeName, clusterName := helpers.K8s1, "default"
+			cmd = fmt.Sprintf("openssl s_client -connect localhost:4244 -showcerts -servername %s.%s.hubble-grpc.cilium.io | openssl x509 -outform PEM", nodeName, clusterName)
```

Comment:
Is `localhost:4244` static, as in it won't be modified? Would it be worth parameterizing this, or is it safe to assume?

Reply:
I would say it is safe to assume, but we can take a middle ground and use the constant representing the default value.